src/helm/benchmark/scenarios/oasst1_scenario.py

```diff
@@ -0,0 +1,106 @@
+from typing import List, Dict, Any
+from datasets import load_dataset, Dataset
+
+from .scenario import CORRECT_TAG, Reference, Scenario, Instance, Input, TRAIN_SPLIT, VALID_SPLIT, Output
+
+
+class Oasst1Scenario(Scenario):
```

Comment:
Perhaps we should call this `OpenAssistantScenario` since if they release subsequent datasets and are in the same format, we can still reuse this class.